presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -417,6 +417,30 @@ public void testShowCreateForPartitionedTablesWithDataCompression()
                         ")");
     }
 
+    @Test
+    public void testShowCreateForIndexedNonPartitionedTablesWithDataCompression()
+    {
+        sqlServer.execute("CREATE TABLE IndexedTable (\n" +
+                "   key1 BIGINT NOT NULL,\n" +
+                "   key2 BIGINT NOT NULL,\n" +
+                "   key3 BIGINT NOT NULL,\n" +
+                "   key4 BIGINT NOT NULL,\n" +
+                "   key5 BIGINT NOT NULL,\n" +
+                "   CONSTRAINT PK_IndexedTable PRIMARY KEY CLUSTERED (key1),\n" +
+                "   CONSTRAINT IX_IndexedTable UNIQUE (key2, key3),\n" +
+                "   INDEX IX_MyTable4 NONCLUSTERED (key4, key5))\n" +
+                "   WITH (DATA_COMPRESSION = PAGE)");
+
+        assertThat((String) computeActual("SHOW CREATE TABLE IndexedTable").getOnlyValue())
+                .matches("CREATE TABLE \\w+\\.\\w+\\.indexedtable \\Q(\n" +
```

Comment:
It's harder to compare text with regexp. 
It's easier to compare text with text.

Please consider 
`.matches` -> `isEqualTo`, 
`\\w+\\.\\w+\\.` -> `getSession().getCatalog().orElseThrow() + "." + getSession().getSchema().orElseThrow() + "." + `
May be extract common sql part.

My version is here https://github.com/starburstdata/starburst-presto/pull/1870/files#diff-43bde5e16ff97c2e1a6bc5ff0861404572ef8c0db16580d69d4d544610b4a23dR253